app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -58,13 +52,6 @@ class ReportsScreen(context: Context, attrs: AttributeSet) : FrameLayout(context
     webView.settings.javaScriptEnabled = true
 
     context.injector<Injector>().inject(this)
-
-    bindUiToController(
-        ui = this,
-        events = events,
```

Comment:
We could also remove emission of the `ScreenCreated` event now that it is no longer in use.